src/sockperf.cpp

```diff
@@ -3373,19 +3384,19 @@ int bringup(const int *p_daemonize) {
                 log_err("Failed to allocate memory with malloc()");
                 rc = SOCKPERF_ERR_NO_MEMORY;
             } else {
-                /* create a socket */
+                    /* create a socket */
                 if ((curr_fd = (int)socket(tmp->server_addr.ss_family, tmp->sock_type, 0)) <
                     0) { // TODO: use SOCKET all over the way and avoid this cast
-                    log_err("socket(AF_INET4/6, SOCK_x)");
+                    log_err("socket(AF_INET4/6 /AF_UNIX, SOCK_x)");
                     rc = SOCKPERF_ERR_SOCKET;
                 } else {
                     if ((curr_fd >= MAX_FDS_NUM) ||
-                        (prepare_socket(curr_fd, tmp.get()) ==
-                         (int)INVALID_SOCKET)) { // TODO: use SOCKET all over the way and avoid
-                                                 // this cast
-                        log_err("Invalid socket");
-                        close(curr_fd);
-                        rc = SOCKPERF_ERR_SOCKET;
+                    (prepare_socket(curr_fd, tmp.get()) ==
+                    (int)INVALID_SOCKET)) { // TODO: use SOCKET all over the way and avoid
+                                            // this cast
+                    log_err("Invalid socket");
+                    close(curr_fd);
+                    rc = SOCKPERF_ERR_SOCKET;
```

Comment:
looks like this code isn't properly formatted

Reply:
Done.